rasa/nlu/training_data/message.py

```diff
@@ -42,22 +44,18 @@ def __init__(
             self.output_properties = output_properties
         else:
             self.output_properties = set()
+        self.output_properties.add(TEXT)
```

Comment:
Why can't we do it as for the other output properties?

Reply:
Because we want it to always include `text` in the output but if no output properties were passed explicitly it won't.  Previously, it was included because `text` was added separately from `self.text` .